recipe/session/recipe.go

```diff
@@ -186,6 +186,10 @@ func (r *Recipe) getAllCORSHeaders() []string {
 func (r *Recipe) handleError(err error, req *http.Request, res http.ResponseWriter) (bool, error) {
 	if defaultErrors.As(err, &errors.UnauthorizedError{}) {
 		supertokens.LogDebugMessage("errorHandler: returning UNAUTHORISED")
+		unauthErr := err.(errors.UnauthorizedError)
+		if unauthErr.ClearCookies != nil && *unauthErr.ClearCookies {
+			clearSessionFromCookie(r.Config, res)
```

Comment:
this should happen in the default onUnauthorised implementation - not here (in here: https://github.com/supertokens/supertokens-golang/blob/clear-cookies-issue/recipe/session/utils.go#L137)

Reply:
Didn't move it because it would be a breaking change.